### PR TITLE
OE-8: Mai Finance Subgraph

### PR DESCRIPTION
**Pull request by miklt**
Fixes #OE-8

Subgraph for Mai Finance yield instruments known as camTokens (compounded aave market tokens) including the following fields:
```
id
blockNumber
blockTimestamp
aTokenAddress
aTvl
decimals
symbol
totalLiquidity
aEmissionPerSecond
address
// for the underlying reserves in AAVE Lending Market
liquidityIndex 
liquidityRate
stableBorrowRate
variableBorrowIndex
variableBorrowRate
```
Obs:
aTvl is the number of aave market tokens (amTokens) linked to the camToken.
aEmissionPerSecond is expressed in WMatic wei units.

#### PR Checklist

- [x] Tests
https://thegraph.com/hosted-service/subgraph/miklt/megagraph-mai-finance?version=pending
- [x] Documentation
- [x] Changelog entry

### CODE REVIEW

File: src/Mai/mappings/CamAave.ts [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  let value = event.params.value;
+
+  // only interested on mints and burns.
+  if (from != ZERO_ADDRESS && to != ZERO_ADDRESS) {
+    return;
+  }
+
+  if (to == ZERO_ADDRESS) {
+    // burn
+    value = value.neg();
```
Comment (uwe) [inline src/Mai/mappings/CamAave.ts:21]: I think `value` is not used. Please check and clean up (in all files). Maybe more can be removed.

File: src/Mai/mappings/handlers.ts [hunk 2]
```diff
@@ -0,0 +1,85 @@
+import { Address, BigInt, Bytes, log } from "@graphprotocol/graph-ts";
+import { MaiCamTokenData } from "../../../generated/schema";
+import { MaiAToken } from "../../../generated/MaiCamAavecamAAVE/MaiAToken";
+import { MaiLendingPool } from "../../../generated/MaiCamAavecamAAVE/MaiLendingPool";
+import { convertBINumToDesiredDecimals, toAddress } from "../../utils/converters";
+
+import { MaiIncentivesController } from "../../../generated/MaiCamTokencamUSDC/MaiIncentivesController";
+import { ZERO_BD, ZERO_BI } from "../../utils/constants";
+
+let CAM_INCENTIVES_CONTROLLER_ADDRESS: Address = toAddress("0x357D51124f59836DeD84c8a1730D72B749d8BC23");
```
Comment (uwe) [inline src/Mai/mappings/handlers.ts:10]: This can go into the constants file.

**Comment by uwe** [comment]
Good progress, I like it much better than the previous version.

File: src/Mai/mappings/CamAave.ts (hunk 1 above)
Comment (miklt) [reply src/Mai/mappings/CamAave.ts]: As I'm getting the data from the directly from the contract, that variable wasn't used anymore. It's removed now.

File: src/Mai/mappings/handlers.ts (hunk 2 above)
Comment (miklt) [reply src/Mai/mappings/handlers.ts]: Done. On the other hand, I let the DECIMALS constant inside the scope of the `convertBINumToDesiredDecimals()` function that was using it.

**Review by uwe — APPROVED** [review]

**Pull request merged**